README.md

```diff
@@ -171,6 +171,18 @@ lsr_role2collection.py [-h] [--namespace NAMESPACE] [--collection COLLECTION]
                      If sub-role name does not start with the specified value, change
                      the name to start with the value; default to an empty string
 --readme README      Path to the readme file used in top README.md
+--extra-mapping EXTRA_MAPPING
+                     This is a comma delimited list of extra mappings to apply when converting the
+                     files - this converts the given name to collection format with the optional given
+                     namespace and collection.
+                     The format is
+                       "src_name:[[dest_namespace.]dest_collection.]dest_name,\
+                        src_name:[[dest_namespace.]dest_collection.]dest_name,...."
+                     The default for `dest_namespace` is the `--namespace` value,
+                     and the default for `dest_collection` is the `--collection` value.
+                     `src_name` is the name of a role, preferably in `namespace.rolename` format.
+                     If just using `rolename` for `src_name`, and `rolename` is used in places
+                     in the README that you do not want to change.
```

Comment:
"do not want to change, you may have to change the README in another way, not using this script, by using `sed` with a custom regexp."

Reply:
looking at the code - looks like `src_name` is treated as `src_owner.src_rolename`?  Does this mean if the user specifies a `src_name` like `storage` with no `.`, does the code look for `linux-system-roles.storage` and try to convert that to `fedora.system_roles.storage`?  Does it look for both `storage` and `linux-system-roles.storage`?

Reply:
This is an example yml:
```
  roles:
    - name: linux-system-roles.storage
    - name: selinux
    - name: rhel-system-roles.logging
```
These 3 command lines converts the example yml to the same format [0].
`$ python  lsr_role2collection.py --role testrole --extra-mapping "storage:storage,selinux:selinux,rhel-system-roles.logging:logging"`
`$ python  lsr_role2collection.py --role testrole --extra-mapping "storage:storage,linux-system-roles.selinux:selinux,rhel-system-roles.logging:logging"`
`$ python  lsr_role2collection.py --role testrole --extra-mapping "linux-system-roles.storage:storage,linux-system-roles.selinux:selinux,rhel-system-roles.logging:logging"
[0]
```
  roles:
    - name: fedora.linux_system_roles.storage
    - name: fedora.linux_system_roles.selinux
    - name: fedora.linux_system_roles.logging
```

Reply:
> This is an example yml:
> 
> ```
>   roles:
>     - name: linux-system-roles.storage
>     - name: selinux
>     - name: rhel-system-roles.logging
> ```
> 
> These 3 command lines converts the example yml to the same format [0].
> `$ python lsr_role2collection.py --role testrole --extra-mapping "storage:storage,selinux:selinux,rhel-system-roles.logging:logging"`
> `$ python lsr_role2collection.py --role testrole --extra-mapping "storage:storage,linux-system-roles.selinux:selinux,rhel-system-roles.logging:logging"`
> `$ python lsr_role2collection.py --role testrole --extra-mapping "linux-system-roles.storage:storage,linux-system-roles.selinux:selinux,rhel-system-roles.logging:logging"
> [0]
> 
> ```
>   roles:
>     - name: fedora.linux_system_roles.storage
>     - name: fedora.linux_system_roles.selinux
>     - name: fedora.linux_system_roles.logging
> ```

What if you had this?
```yaml
 roles:
    - name: storage
    - name: linux-system-roles.storage
    - name: selinux
    - name: rhel-system-roles.logging
```
?

Reply:
Ok. There is a bug. :) This is converted to `- name: fedora.linux_system_roles.storage`, as well.
```
  roles:
    - name: something-else.storage
```